### DISCUSSION TITLE
Can you set operation size_maximum on import?

### DISCUSSION THREAD

**Discussion by fatLittleDog** [frePPLe/frepple#387] [Q&A] [answered]
Hi!  I am importing my operations into Frepple from an excel sheet.  Is there a way to specify the size_maximum in the import?  I have tried a few different column headers, but they are not recognized.  Trying to avoid having to go into the advanced settings of each operation to set this data point.

Thanks,

Peter

**Comment by jdetaeye** [comment, marked as answer]
Of course you can.
Use the wrench icon to customize the operation list report and display the "size_maximum" field in the layout - see https://frepple.org/docs/current/user-interface/getting-around/customizing-a-report.php
You can export and import excel & csv files with that field included.  You can export the data to get a template for re-importing that data.
